src/apps/filters/h3.c

```diff
@@ -110,6 +116,41 @@ bool latLngToCellCmd(int argc, char *argv[]) {
     return true;
 }
 
+bool cellToBoundaryCmd(int argc, char *argv[]) {
+    Arg cellToBoundaryArg = {
+        .names = {"cellToBoundary"},
+        .required = true,
+        .helpText =
+            "Convert an H3 cell to an array of latitude/longitude coordinates "
+            "defining its boundary",
```

Comment:
Does this description still make sense if we're outputting WKT?

Reply:
Kinda? But I'll change it now to be more precise.